pkg/reconciler/route/table_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Knative Authors.
+Copyright 2018 The Knative Authors
```

Comment:
<!--copyright-bot-->
Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

Reply:
by mistake `.` has been removed but for this PR no changes in the license content so reverted back by adding `.`